### ISSUE TITLE
Upgrading my graphics card

### ISSUE THREAD

**Issue by andrewacosta117** [ValveSoftware/SteamOS#535]
#### Your system information

* Steam client version: 4.1.0-0.steamos.25-amd64
* SteamOS version: 2.0
* Opted into Steam client beta?: No
* Opted into SteamOS beta?: No
* Have you checked for updates in Settings > System?: No

#### Please describe your issue in as much detail as possible:
When I uploaded SteamOS onto my computer I had an old Radeon ATI HD5770 graphics card and it worked fine, but I was not getting the preformance I wanted. I bought a NVIDIA GTX1070 FE because I heard that NVIDIA plays better on Linux. I simply swapped the cards and started the computer. After the bios screen all that would show up is a giant blinking mouse curser. Am I missing drivers? How can I install them if I can't get into the computer? Please help I don't want to go back to Windows.

#### Steps for reproducing this issue:

1. Swap graphics card
2. turn on computer
3. black screen with blinking curser

**Comment by Tele42** [comment]
Hello @andrewacosta117, I'm not sure, but I think you need to switch back to your previous video card, opt into the steamos beta, make sure it updated, then switch to your new card. You need at least SteamOS Brewmaster update 2.83 in order to have a chance of supporting such a new card as support for the GTX 1070 was added in the newest nvidia driver release.

**Comment by andrewacosta117** [comment]
@Tele42 Thanks for the advice. I reinstalled my old graphics card and attempted to opted into steamOS beta and I thought I updated to 2.83 using `apt-get update` but no such luck. (still very new to linux). But what I did do was go into Settings > Details and hit the "Check for updates" ... 77 updates avaliable... *facepalm* and I noticed some NVIDIA drivers like 367 which is NVIDIAs current Linux driver. After updates install shut down, swap cards, turn on, and all good.

If anyone new to Linux is reading this make sure you go to (in desktop mode) Settings > Details and click "Check for updates"
